packages/hebao_v1/contracts/modules/security/GuardianModule.sol

```diff
@@ -33,11 +33,11 @@ contract GuardianModule is SecurityModule
     uint public pendingPeriod;
     uint public confirmPeriod;
 
-    mapping (address => mapping(address => uint)) public pendingAdditions;
+    mapping (address => mapping(address => mapping(uint => uint))) public pendingAdditions;
     mapping (address => mapping(address => uint)) public pendingRemovals;
 
-    event GuardianAdditionPending   (address indexed wallet, address indexed guardian, uint confirmAfter);
-    event GuardianAdded             (address indexed wallet, address indexed guardian);
+    event GuardianAdditionPending   (address indexed wallet, address indexed guardian, uint types, uint confirmAfter);
```

Comment:
recommend renaming 'types' to type as it will only be used to represent one type.